src/components/volunteer/VolunteerGroup.tsx

```diff
@@ -68,6 +73,7 @@ export const VolunteerGroup = ({
   const [isGroupMember, setIsGroupMember] = useState<boolean | undefined>();
   const [isGroupOwner, setIsGroupOwner] = useState(false);
   const [isGroupApplicant, setIsGroupApplicant] = useState(false);
+  const [groupAdmins, setGroupAdmins] = useState<Array<string>>([]);
```

Comment:
`string`? there will be numbers in the array, right? and the default value could be the `id` of the `owner` prop.

Reply:
Oh, yeah. I'll fix that. Why wouldn't the default value be an empty array?

Reply:
because with `owner.id` we already have some admin id to provide for group admins instead of nothing. so the first render cycle already has that information. if there is a group with only one member, which is the owner in that case, we can avoid filtering and mapping to check for further group admin to set the state with the one id of that particular owner/admin.

Reply:
eade8fd